src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -724,6 +726,11 @@ static QString slurTieLineStyle(const SlurTie* s)
     if (!lineType.isEmpty()) {
         rest = QString(" line-type=\"%1\"").arg(lineType);
     }
+    if (s->slurDirection() != engraving::DirectionV::AUTO) {
```

Comment:
Can you add ` || configuration()->musicxmlExportLayout()` here (like #17842 does)?